src/index.js

```diff
@@ -1,4 +1,5 @@
 import {NativeModules, NativeEventEmitter, Platform} from 'react-native';
+import _ from 'underscore';
```

Comment:
I was thinking that we don't need this dependency for this project. We can disable this rule in the eslint config. @luacmartins What do you say?

Reply:
While I hate to add this dependency for a single usage of map, I think that it's best if we stay consistent with our other repos. So I'd say that we keep underscore here.